specs/mediawiki/v1/content.yaml

```diff
@@ -840,6 +903,9 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/parsoid/transform/wikitext/to/html{/title}{/revision}
+        body:
+          wikitext: '{wikitext}'
+          bodyOnly: '{bodyOnly}'
```

Comment:
Unfortunately, VE uses `body` for compatibility with systems that run Parsoid, but not RESTBase, cf. https://github.com/wikimedia/mediawiki-extensions-VisualEditor/blob/master/ApiVisualEditor.php#L181